src/displayScores.js

```diff
@@ -0,0 +1,16 @@
+import appendScore from './appendScore';
+
+const board = document.querySelector('.leaderboard');
+
+async function displayScores() {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/jXwsUKYAzNn8VCQ6q31T/scores');
```

Comment:
- Optional
I would recommend you display the scoreboard when the user visits the application, instead of fetching the scoreboard when the refresh button is clicked.